protobuf/transform_test.go

```diff
@@ -0,0 +1,282 @@
+package protobuf
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/src-d/proteus/resolver"
+	"github.com/src-d/proteus/scanner"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestToLowerSnakeCase(t *testing.T) {
```

Comment:
It will fail with `fooBAR` for example. (result:` foo_b_a_r`. I think the expected result should be: `foo_bar`)